contracts/utils/Checkpoints.sol

```diff
@@ -15,21 +16,233 @@ import "./math/SafeCast.sol";
  * _Available since v4.5._
  */
 library Checkpoints {
```

Comment:
I think we should include a comment in the source code saying "the following is procedurally generated by the script X". Not in the natspec docs, just as a comment for anyone reading this file directly. This applies to SafeCast as well.

Reply:
Resolved in the generation file